sys/include/suit/cbor.h

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (C) 2018 Freie Universit�t Berlin
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_suit SUIT manifest parsers
+ * @ingroup     sys
+ * @brief       SUIT manifest parser for CBOR based manifests
+ *
+ * This is a simple suit manifest parser for RIOT. The high level assumption is
+ * that the raw manifest data is stored in a buffered location where raw values
+ * or strings can be left during the lifetime of the suit_manifest_t struct.
+ * This assumption is valid in the case where a (network based) transfer
+ * mechanism is used to transfer the manifest to the node and an intermediate
+ * buffer is necessary to validate the manifest.
+ *
+ * The parser is based on draft version 1 of the specification, restricted to
+ * handling CBOR based manifests.
+ *
+ * @see https://tools.ietf.org/html/draft-moran-suit-manifest-01
```

Comment:
Maybe add the `@experimental` tag as well, since this is based on a draft?

Reply:
see

https://github.com/RIOT-OS/RIOT/blob/e288c9b23fcbfc6744ca377aa3a27cbb0ad19d9f/doc/doxygen/riot.doxyfile#L231

Reply:
Done